src/Util/ClassSourceManipulator.php

```diff
@@ -358,6 +367,17 @@ public function addProperty(string $name, array $annotationLines = [], $defaultV
         $this->addNodeAfterProperties($newPropertyNode);
     }
 
+    public function addAttributeToClass(string $attributeClass, array $options): void
```

Comment:
Adding a class level attribute that requires an aliased use statement, e.g. ` use Doctrine\Mapping as ORM` is not possible in this scope without refactoring the `ClassSourceManipulator::addUseStatementIfNecessary()` method. A task which is not required by our current use case and should be handled in a separate PR.